frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/viewport-feature-manager.ts

```diff
@@ -61,15 +61,9 @@ export class ViewportFeatureManager
         );
         this.layer.setStyle(this.style);
     }
-    private readonly modifyHelper = new ModifyHelper();
 
     private readonly style = new Style({
-        geometry(thisFeature) {
-            const modifyGeometry = thisFeature.get('modifyGeometry');
-            return modifyGeometry
-                ? modifyGeometry.geometry
-                : thisFeature.getGeometry();
-        },
+        fill: undefined,
```

Comment:
```suggestion
```

I worked on a branche where #603 was already partially implemented -> this should not be changed, yet